internal/services/network/network_service_tags_data_source.go

```diff
@@ -42,6 +43,22 @@ func dataSourceNetworkServiceTags() *pluginsdk.Resource {
 					Type: pluginsdk.TypeString,
 				},
 			},
+
+			"ipv4_cidr": {
```

Comment:
was updating the PR title and realized these should be
```suggestion
			"ipv4_cidrs": {
```

Reply:
Yep makes sense, updated the name to cidrs.